features/step_definitions/show_geotrust_ssl_image_steps.rb

```diff
@@ -0,0 +1,7 @@
+Given(/^I am on the search page with the map "([^"]*)"$/) do |page|
+  visit page
+end
+
+Then("I should see the GeoTrust SSL certificate Image") do
+  expect(page).to have_xpath("//img[contains(@src,'RapidSSL_SEAL-90x50')]")
+end
```

Comment:
I don't think this change merits a complete new feature file.  I would suggest adding the check on the image to an existing feature file, and to an existing scenario.  Perhaps `log_in.feature`, first scenario.

Reply:
Agree that this can be covered in a scenario (or scenarios) in existing features.
Per thesuss's suggestion to have this on views that have to do with _payment,_ I'd put the scenarios in those features.

Reply:
@patmbolger @weedySeaDragon Agree on the incorporation into another feature file. When I started the feature files' structure seemed a bit daunting and I just wanted to get smth done. Was hoping to get a pointer like this, so thank you. =)

Reply:
Yup -- we definitely don't have any guidance for that; there was no way of knowing that ahead of time. :-)

Reply:
@weedySeaDragon Yes, agree it should go in those features. I only found the steps so far, so I will have a look through all feature files both to find the right scenario and familiarize myself with things. :)

Reply:
I put it in the `member_pays_membership` feature instead, keeping the steps definitions in a separate file for now.